src/client/components/SampleComponent/SlideshowImageComponent/slideshowImageComponent.stories.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import SlideshowImageComponent from './SlideshowImageComponent';
+
+export default { title: 'Image Component' };
```

Comment:
Can't call it 'image component'. Then you might as well call your website 'website' or your variable 'variable'

Reply:
Yes. I changed it into Avatar Component since this component loads different avatars. I will push the changed code tomorrow.